treeherder/log_parser/intermittents.py

```diff
@@ -17,22 +17,23 @@ def _check_and_mark_infra(current_job, job_ids, push_ids):
     # look for all jobs in pushids matching current_job.job_type.name
     # if older are failing for "infra", then ensure same job is passing
     #  if so mark as intermittent
-    extra_jobs = JobLog.objects.filter(
-        job__push__id__range=(push_ids[-1], push_ids[0]),
-        job__push__repository__id=current_job.repository.id,
-        job__job_type__name=current_job.job_type.name,
-        job__failure_classification_id__in=[1, 6],
-        status__in=(1, 2, 3),  # ignore pending
-        job__result__in=[
+    extra_jobs = Job.objects.filter(
+        push__id__range=(push_ids[-1], push_ids[0]),
+        repository__id=current_job.repository.id,
+        job_type__name=current_job.job_type.name,
+        failure_classification_id__in=[1, 6],
+        job_log__status__in=[1, 2, 3],  # ignore 0 == PENDING
```

Comment:
The negation (not 0) should be better.

Reply:
actually status=2 is FAILED, but this is due to failing to read the log file, not test failed, etc.; so I need status to be either 1 or 3.